src/sources/socket/mod.rs

```diff
@@ -383,6 +381,53 @@ mod test {
         SourceSender,
     };
 
+    fn get_gelf_payload(message: &str) -> String {
+        serde_json::to_string(&json!({
+            "version": "1.1",
+            "host": "example.org",
+            "short_message": message,
+            "timestamp": 1234567890.123,
+            "level": 6,
+            "_foo": "bar",
+        }))
+        .unwrap()
+    }
+
+    fn create_gelf_chunk(
+        message_id: u64,
+        sequence_number: u8,
+        total_chunks: u8,
+        payload: &[u8],
+    ) -> Bytes {
+        const GELF_MAGIC: [u8; 2] = [0x1e, 0x0f];
```

Comment:
Hardcode the GELF_MAGIC, although we have it in lib/codecs/src/decoding/framing/chunked_gelf.rs#L15, I found weird exporting that constant for testing purposes..